docs/rebuild_admin.rst

```diff
@@ -6,7 +6,7 @@ Rebuilding an *Admin Workstation* USB
           rebuild your *Admin Workstation*, please contact FPF through the
           `SecureDrop Support Portal`_.
 
-.. _SecureDrop Support Portal: https://securedrop-support.readthedocs.io/en/latest/
+.. _SecureDrop Support Portal: https://support-docs.securedrop.org/en/latest/en/latest/
```

Comment:
This is an invalid URL that 404s (notice repetition of `/en/latest`)

Reply:
Good catch, thank you! I thought I had caught all of those, I missed them when proof reading : /